### ISSUE TITLE
Sync stopped working

### ISSUE THREAD

**Issue by SeeThisIsMe** [jsetton/alexa-ourgroceries-sync#13]
Is this skill still working? This skill used to work fine for a long time, but recently (maybe a few weeks ago) it stopped working. 

Now when giving an Alexa command to add something to the list, it only adds directly to the Alexa shopping list, OurGroceries does not get the update. I deleted and redeployed the skill again but it is still the same - OurGroceries doesn't get the update when adding something and the Alexa list does.

Thanks.

**Comment by SeeThisIsMe** [comment]
Great - thanks!

**Comment by jsetton** [comment]
The Alexa Developer support team came back stating the household list skill [requires account linking](https://developer.amazon.com/en-US/docs/alexa/smapi/steps-to-create-a-list-skill.html#alexa-lists-access) and they have released a bug fix that removed the ability for such skill to receive `AlexaHouseholdListEvent` events when it wasn't setup for such process.

Although I don't remember such requirement being present when I first developed this skill, I have released #16 with an [updated documentation](https://github.com/jsetton/alexa-ourgroceries-sync#login-with-amazon-security-profile) providing instructions on how to enable account linking with Login with Amazon (LWA). Once configured and enabled, I can confirm that the list events are received by the skill again.
